stake-pool/js/package.json

```diff
@@ -15,28 +15,15 @@
   "keywords": [],
   "authors": [
     "Solana Maintainers <maintainers@solana.foundation>",
-    "Lieu Zheng Hong",
+    "Lieu Zheng Hong", 
     "mFactory Team (https://mfactory.ch)"
   ],
-  "homepage": "https://solana.com",
```

Comment:
Should these lines be removed from `package.json`? I think most of these removed fields are good to have in the `package.json` file and probably should be added back (unless there's a reason to remove them that I don't know about).

Reply:
yea, sorry about it .. merge issues ... thank you for the note

Reply:
removed react dep from package json"eslint-plugin-react": "^7.28.0",